code/modules/clothing/spacesuits/rig/modules/modules.dm

```diff
@@ -54,13 +55,37 @@
 
 	var/list/stat_rig_module/stat_modules = new()
 
+/obj/item/rig_module/New()
```

Comment:
It's better to call that in `Initialize()` instead of `New()`

Reply:
This is just bay's outdated proc, moved up becouse i like `New()`/`Destroy()` going right after object definition.